Add LearnWeb3Dao and AlchemyLearn to Tutorial Section (#115)
bkrem/awesome-solidity: README.md

- [ethereum/solidity-examples](https://github.com/ethereum/solidity-examples) - Loose collection of example code.

#### Tutorials

> - [AlchemyLearn](https://university.alchemy.com/) - Fast track your web3 career through courses, projects and code. Powered by [AlchemyPlatform](https://twitter.com/alchemyplatform)

Reviewer: Sorry but not keen to shill Alchemy directly via link on the list. Feel free to provide a more instructional description of what this actually provides or accept if ok for you like this 👍

```suggestion
- [AlchemyLearn](https://university.alchemy.com/) - web3 courses, projects and code from the Alchemy team.
```

- [ExtropyIO/defi-bot](https://github.com/ExtropyIO/defi-bot) - Tutorial for building DeFi arbitrage bots.
- [karmacoma-eth/sending-ether-cheat-sheet](https://gist.github.com/karmacoma-eth/4f206a46dedc6da6808c1ccdef3262d0) - Tips and best practices for sending Ether.
- [LearnXInY](https://learnxinyminutes.com/docs/solidity/) - Learn Solidity in 15 mins (for experienced devs).

> - [LearnWeb3Dao](http://learnweb3.io/) - A free program that will turn you into a web3 developer. Build a Web3 career the quickest way possible.

Reviewer: Also too much of a marketing description, let’s please explain in a sentence what the resource provides
